app/pages/lab/workflows.jsx

```diff
@@ -19,23 +20,47 @@ const WorkflowsPage = (props) => {
     );
   });
 
+  const reorderButton = props.reorder ?
+    <button type="button" data-button="reorderWorkflow" onClick={props.toggleReorder}>List view</button> :
+    <button type="button" data-button="reorderWorkflow" onClick={props.toggleReorder}>Reorder view</button>;
+  const meta = props.workflows.length ? props.workflows[0].getMeta() : {};
+
   return (
     <div>
-      <header className="form-label">Workflows</header>
+      <h1 className="form-label">Workflows</h1>
 
       <p>A workflow is the sequence of tasks that you’re asking volunteers to perform.</p>
-      <p> An asterisk (*) denotes a default workflow. </p>
-      <p> If you have multiple workflows you can rearrange the order in which they are listed on your project's front page by clicking and dragging on the left gray tab next to each workflow title listed below. </p>
+      <p>An asterisk (*) denotes a default workflow.</p>
+      <p>If you have multiple workflows you can rearrange the order in which they are listed on your project's front page by clicking the reorder view button and then clicking and dragging on the left gray tab next to each workflow title listed below.</p>
+      <p>{reorderButton}</p>
+      
+      {props.reorder &&
+        <DragReorderable tag="ul" className="nav-list" items={props.workflows} render={renderWorkflow} onChange={props.handleWorkflowReorder} />}
 
-      <DragReorderable tag="ul" className="nav-list" items={props.workflows} render={renderWorkflow} onChange={props.handleWorkflowReorder} />
+      {(!props.reorder && props.workflows.length > 0) &&
+        <div>
+          <ul className="nav-list">
+            {props.workflows.map(workflow => renderWorkflow(workflow))}
+          </ul>
+          <hr />
+          <Paginator
+            className="talk"
```

Comment:
wouldn't have noticed if hadn't pointed out in different PR, but do we want to avoid using the "talk" class outside of Talk? not sure.